Original transcript improvements

georgebrock opened the pull request:
* Keep the “previous” and “next” buttons in the same place as you scroll (#58)
* Link back to the transcript view (#53)
* Alt text on the transcript page image

jaylett commented:
LGTM other than note about commenting.

jaylett commented:
Thank you github, I certainly wanted non-inline PR comments when I commented on individual commits, because they are infinitely easier to read…

georgebrock commented:
@jaylett I've made the SCSS more explicit, let me know if it's an improvement.

(I've also added the `rel=alternate` you suggested, and rebased onto master)
